.rubocop.yml

```diff
@@ -20,3 +20,7 @@ Metrics/MethodLength:
 
 Style/RescueModifier:
   Enabled: false
+
+ClassLength:
+  Exclude:
+    - 'features/support/pages/appointment_summary_page.rb'
```

Comment:
We're already excluding a rule from [app/models/appointment_summary.rb](https://github.com/guidance-guarantee-programme/output/blob/bc53223ad624497c1953d55795410d7f9e804e69/app/models/appointment_summary.rb#L88) in the source; do we want to put all excludes in this top-level file or in the individual files themselves?